src/components/AddPeopleAndRoles/OrgPerson.vue

```diff
@@ -0,0 +1,533 @@
+<template>
+  <div>
+    <confirm-dialog ref="reassignCPDialog" attach="#addEditPersonContainer"/>
+    <v-expand-transition id="addEditPersonContainer">
+      <ul class="list add-person">
+        <li class="add-person-container">
+          <div class="meta-container">
+            <label class="add-person-header" v-if="isPerson">
+              <span v-if="activeIndex===-1">Add Person</span>
+              <span v-else>Edit Person</span>
+            </label>
+            <label class="add-person-header" v-if="isOrg">
+              <span v-if="activeIndex===-1">Add Corporation or Firm</span>
+              <span v-else>Edit Corporation or Firm</span>
```

Comment:
Per mockup, please add in handling for BCOMP vs COOP (ie, different labels).

Reply:
I can't actually find what you mean in regards to these two comments.

Reply:
The labels are supposed to be different depending on entity type.

Reply:
Found the bit in the design, updated 👍